services/spar/src/Spar/Scim/User.hs

```diff
@@ -442,7 +442,7 @@ to a single `externalId`.
 -}
 assertUserRefUnused :: SAML.UserRef -> Scim.ScimHandler Spar ()
 assertUserRefUnused userRef = do
-  mExistingUserId <- lift $ wrapMonadClient (Data.getSAMLUser userRef)
+  mExistingUserId <- lift $ getUser userRef
```

Comment:
`getUser` only reports conflicts in the same team.

This might work for now if we have a rule that each IdP can only be "used" by one team, but it seems fragile and I don't think anyone will remember to fix it if we lift that rule somehow.

(To be honest I'm not even 100% sure we have that rule, off the top of my head.)

Reply:
if we add the same idp to two teams, implicit user creation breaks immediately: we won't know any more where to create the user.  and i don't expect implicit user creation via saml to go away ever.

more importantly: why do you think cross-team conflicts are not reported?y  i think they are.

but if you know of a good place where i can add a comment that idps can only live in one team, i'd like to do that.

Reply:
> why do you think cross-team conflicts are not reported

Oh, this is interesting. Even though the `getUser` method is defined in `Spar.Scim.User`, it's not in scope – instead `getUser` from `Spar.App` is used.

I'm marking this as resolved.